backend/routers/buildings.js

```diff
@@ -0,0 +1,31 @@
+const { Router } = require('express');
+
+const router = Router();
+
+const myBuildings = {
+  buildings: [
+    {
+      id: 1,
+      type: 'townhall',
+      level: 1,
+      hp: 1,
+      started_at: 12345789,
+      finished_at: 12399999,
+    }, {
+      id: 2,
+      type: 'farm',
+      level: 1,
+      hp: 1,
+      started_at: 12345789,
+      finished_at: 12399999,
+    },
+  ],
+};
+
+function getBuildings(req, res) {
+  return res.status(200).send(myBuildings);
+}
+
+router.get('/buildings', getBuildings);
```

Comment:
if we go with app.use('/kingdom/buildins', buildings); in the App.js, then here the '/' is enough
```suggestion
router.get('/', getBuildings);
```